echemdb/data/cv/entry.py

```diff
@@ -375,53 +341,104 @@ def __repr__(self):
 
             >>> entry = Entry.create_examples()[0]
             >>> entry
-            Entry('alves_2011_electrochemistry_6010_p2_f2a_solid')
+            Entry('alves_2011_electrochemistry_6010_p2_f1a_solid')
 
         """
         return f"Entry({repr(self.identifier)})"
 
-    def plot(self, xunit=None, yunit=None):
+    def _verify_field_name(self, field_name):
+        """Verify if a given field name exists in the resources field names.
+
+        EXAMPLES::
+
+            >>> entry = Entry.create_examples()[0]
+            >>> entry._verify_field_name('E')
+            'E'
+
+        For a field with name `x` that does not exist::
+
+            >>> entry = Entry.create_examples()[0]
+            >>> entry._verify_field_name('x')
+            Traceback (most recent call last):
+            ...
+            ValueError: None of the axes is named 'x'.
+
+        """
+        if field_name not in self.package.get_resource('echemdb').schema.field_names:
+            raise ValueError(f"None of the axes is named '{field_name}'.")
+        return field_name
+
+    def plot(self, x_label='E', y_label='j'):
         r"""
-        Return a plot of the data in this data package in SI units.
+        Return a plot of the 'echemdb resource' in this entry. 
+        The default plot is a cyclic voltammogram ('j vs E').
+        When `j` is not defined `I` is used instead.
 
         EXAMPLES::
 
             >>> entry = Entry.create_examples()[0]
             >>> entry.plot()
             Figure(...)
 
+        # TODO: Remove or adaopt the following doctests.
         The plot can also be returned with the axis units of the original figure::
 
-            >>> entry = Entry.create_examples()[0]
-            >>> entry.plot(xunit='original', yunit='original')
+            # >>> entry = Entry.create_examples()[0]
+            # >>> entry.plot(xunit='original', yunit='original')
             Figure(...)
 
-        The plot can also be returned with custom axis units, where 
-        `xunit` should be convertible to `V` and 
+        The plot can also be returned with custom axis units, where
+        `xunit` should be convertible to `V` and
         `yunit` convertible to `A` or `A / m2`.::
 
-            >>> entry = Entry.create_examples()[0]
-            >>> entry.plot(xunit='mV', yunit='uA / cm2')
+            # >>> entry = Entry.create_examples()[0]
+            # >>> entry.plot_cv(xunit='mV', yunit='uA / cm2')
             Figure(...)
 
         """
         import plotly.graph_objects
 
-        xunit = self.x_unit(xunit)
-        yunit = self.y_unit(yunit)
+        def catching_label(label):
+            r"""Verifies that the labels exit. When a current density exists
```

Comment:
Something went wrong in this docstring. Also, please add a doctest.

Should this be a hidden `_method`?